### PR TITLE
스테이징 환경에서 구분선 1, 2, 3 스타일이 보이지 않은 문제 고치기

### PR DESCRIPTION
**Pull request by x86chi**

### CODE REVIEW

**Comment by x86chi** [comment]
Current dependencies on/for this PR:
* main
  * **PR #455** <a href="https://app.graphite.dev/github/pr/penxle/penxle/455?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/455?utm_source=stack-comment).

**Comment by x86chi** [comment]
✅

File: apps/penxle.com/src/routes/editor/Footer.svelte [hunk 1]
```diff
@@ -6,6 +6,7 @@
   let open = false;
 </script>
 
+œ
```
Comment (Seohyun-Roh) [inline apps/penxle.com/src/routes/editor/Footer.svelte:9]: 요건 뭔가요..??

**Review by Seohyun-Roh — CHANGES_REQUESTED** [review]

File: apps/penxle.com/src/routes/editor/Footer.svelte (hunk 1 above)
Comment (x86chi) [reply apps/penxle.com/src/routes/editor/Footer.svelte]: 아... 따른 수정 사항이 들어갔네요 😓 뺐습니다!

**Review by Seohyun-Roh — APPROVED** [review]

**Pull request merged**